layouts/partials/engineering-simplified.html

```diff
@@ -0,0 +1,42 @@
+<section class="mt-10">
+    <div class="pb-10">
+        <h2 class="font-bold text-xl sm:text-2xl lg:text-2xl">Data Engineering Simplified</h2>
```

Comment:
`sm:text-2xl` is applied to all screens at that size and larger, so `lg:text-2xl` shouldn't be needed here.

Reply:
Good catch , we should review it in the contact form too. I just copy what I did there.

Reply:
I searched the files and didn't see the repeated style listed anywhere else.